src/sympc/tensor/mpc_tensor.py

```diff
@@ -1125,6 +1130,32 @@ def ne(self, other: "MPCTensor") -> "MPCTensor":
         other = self.__check_or_convert(other, self.session)
         return 1 - self.eq(other)
 
+    def xor(self, other) -> "MPCTensor":
+        """XOR operator.
+
+        Args:
+            other (MPCTensor): MPCTensor to find xor.
```

Comment:
I think the type annotation should also include Union[int,torch.Tensor,MPCTensor].

Reply:
Falcon has defined xor for public operations, maybe we could use the same formula for private ops like MPCTensor1 ^ MPCTensor2 , This would be useful in bit injection PR,as we could directly call xor.

Reply:
https://github.com/OpenMined/SyMPC/blob/b230edd7d7792d0804f951dd7c8e81ae533d7dc2/src/sympc/protocol/aby3/aby3.py#L280

Reply:
[Here](https://github.com/OpenMined/PySyft/pull/4474/files#diff-5c9f2f84a6d938eeea9e386ce5c519f087f5d606f22958a4556e1f0fccaf4230) is the implementation we used in the previous Pysyft

Reply:
@gmuraru as @rasswanth-s  asked we are trying it for other ring sizes too/